.travis.yml

```diff
@@ -1,26 +1,25 @@
 language: csharp
-sudo: false
+sudo: required
 dist: trusty
-dotnet: 1.0.0
+mono: none
+dotnet: 1.0.1
+env:
+  global:
+    - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+    - DOTNET_CLI_TELEMETRY_OPTOUT: 1   
+    - TEMP: "$TRAVIS_BUILD_DIR/.temp"
 addons:
```

Comment:
Then it fails ... I twill try again once I get greens. linux is green right now, waiting for macos.